gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/oauth2/service/token/Token.java

```diff
@@ -86,6 +87,10 @@ public abstract class Token implements Serializable {
      */
     public static final String UPGRADED = "upgraded";
 
+    public static List<String> getStandardParameters(){
+        return List.of(ACCESS_TOKEN, TOKEN_TYPE, EXPIRES_IN, REFRESH_TOKEN);
```

Comment:
issue: at least id_token is missing, please check in the OpenId specification if some other attributes can be present

Reply:
adde id_token and checked the documentation. Only id_token was missing.

Reply:
Why not adding id_token here  ?

Reply:
Because this is not part of the Token class . In my feeling getStandardParameter should return only fields that are in this class.